hoomd_organics/base/simulation.py

```diff
@@ -83,12 +84,30 @@ def __init__(
         self.integrator = None
         self._dt = dt
         self._reference_values = dict()
+        self._reference_values = reference_values
         self._integrate_group = hoomd.filter.All()
         self._wall_forces = dict()
         self._create_state(self.initial_state)
         # Add a gsd and thermo props logger to sim operations
         self._add_hoomd_writers()
 
+    @classmethod
+    def from_system(cls, system, **kwargs):
+        """Initialize a simulation from a system object."""
+
+        return cls(
+            initial_state=system.state,
+            forcefield=system.hoomd_forcefield,
+            reference_values=system.reference_values,
+            **kwargs,
+        )
+
+    @classmethod
+    def from_init_state_forces(cls, init_state, forcefield, **kwargs):
```

Comment:
I think the name here should be a little more descriptive. Something like `from_snapshot_forces`

Reply:
hmmm, but you an also pass in a gsd file..so maybe snapshot doesn't make sense here.

Reply:
how about something like `resume_simulation`?

Reply:
While it is the method someone would use when restarting/resuming a simulation, I don't think its the only use case for that, so it might be confusing if we name it that.

Reply:
I see. So in that case I think `from_snapshot_forces` would be a better choice cause a gsd file also contains a snapshot of a system.